src/_posts/platform/databases/2000-01-01-access.md

```diff
@@ -154,13 +157,13 @@ Example:
 If the app is in the region `osc-fr1` and if the environment variable is the following:
 
 ```
-SCALINGO_POSTGRESQL_URL=postgresql://user:secret@my-db.postgresql.dbs.appsdeck.eu:30000/my-db
+SCALINGO_POSTGRESQL_URL=postgresql://user:secret@my-db.postgresql.dbs.scalingo.com:30000/my-db
```

Comment:
There are other places in the doc with `appsdeck.eu`. Can you also update these URLs?